internal/pkg/cli/pipeline_status_test.go

```diff
@@ -8,55 +8,356 @@ import (
 	"errors"
 	"fmt"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/mocks"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 	"testing"
 )
 
+var (
+	mockErr             = errors.New("some error")
+	mockApplicationName = "my-app"
+	mockPLName          = "pipeline-dinder-badgoose-repo"
+)
```

Comment:
Nit: You could ostensibly just use the vars defined in the pipeline_show_test.go file, since they appear to be basically the same (See: https://github.com/aws/amazon-ecs-cli-v2/blob/master/internal/pkg/cli/pipeline_show_test.go#L21-L25) --  `var` declarations apply to the whole package (in this case, the `cli` package).

Of course (disclaimer!) global variables are generally not super, but in this case I think it's fine to reuse these.